source/templates.tex

```diff
@@ -2472,9 +2472,11 @@
 \end{codeblock}
 \end{example}
 \begin{note}
+%% Ed. note: Using \iref would cause an overflowing \hbox; changing
```

Comment:
I thought we'd fixed `\iref` so that we *can* break before it, just with a high penalty? :(

Reply:
Except that, when I tried it, it didn't work in this case.

Reply:
Perhaps we could add an optional parameter to `\iref` to specify its `\nolinebreak` penalty. I'm sad that it doesn't "just work" here; it definitely does work in other places, such as [class.dtor]p14, but IIRC turning down the line break penalty any further globally causes TeX to wrap inline references even in places where it's unnecessary to do so.

Reply:
From https://tex.stackexchange.com/questions/94217/what-is-the-difference-between-nobreak-and-nolinebreak it looks like we're probably effectively using `\penalty 301` currently (that's for article, but I'd guess memoir is probably not very different). Maybe we could binary search to find whatever value works in this case and try changing `\iref` to use something like that.